### ISSUE TITLE
Prometheus pod crash loop on long-lasting clusters

### ISSUE THREAD

**Issue by polskikiel** [kyma-project/kyma#6677]
**Description**

Prometheus pod goes crash-loop backoff on long-lasting clusters.
Because of that, we cannot reach the metrics from any pod. 

From weekly cluster:

![image](https://user-images.githubusercontent.com/21159554/71081207-0d930400-218f-11ea-9729-a3324b1ae93e.png)

There was a lot of alerts about that recently: 
![image](https://user-images.githubusercontent.com/21159554/71087368-4d141d00-219c-11ea-8506-eb39b1bb66f0.png)

**Expected result**

Prometheus pod has no restarts 

**Actual result**

Prometheus restarts even a few thousand times per week

**Comment by a-thaler** [comment]
After looking into the cluster it seems that we found that prometheus is down because of a serious problem and we will look into it.

However, the alerts are not related to it, they are just indicating that test pods are down, no relation to the dashboard and unavailability of prometheus itself.

**Comment by suleymanakbas91** [comment]
We realized that monitoring integration tests caused some memory hikes on prometheus. This is fixed by https://github.com/kyma-project/kyma/commit/75cd95fac5c80339ad60c83dbfeae6889f7277b5 and https://github.com/kyma-project/kyma/commit/c3db927190aff4cd4d10f8f7bba4483a96e6bb12.

We also increased the scrape interval for istio service monitor (see https://github.com/kyma-project/kyma/commit/3c8b260f8bcb66e892a4f845823e34363ee734d9), which frees some load from prometheus. 

Let's observe for some more time if all these will be enough to have it consistently stable.
